Netflix/metaflow-ui
# Bug/hide logo

### Requirements for a pull request


### Description of the Change

Hide logo on full page when `HIDE_LOGO` feature flag is set

### Alternate Designs

\-

### Possible Drawbacks

\-

### Verification Process

* Set `HIDE_LOGO` to true
* Navigate to a full page (click on the full page icon next to stdout or stderr on a task page)
* Confirm that no Metaflow icon is shown on the top-left of the page

### Release Notes

* `HIDE_LOGO` for full page
